app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/actionableaddress/ActionableAddressView.kt

```diff
@@ -162,14 +162,16 @@ fun ActionableAddressView(
 
 @Suppress("CyclomaticComplexMethod")
 @Composable
-private fun ActionableAddressView(
+fun ActionableAddressView(
```

Comment:
Why did this needed to become public?

Reply:
To be able to call it with the interface type. Please see AccountCardView and AccountUiItem in WalletViewModel, this was needed to have all the underlying fields initialized in the view model.

Reply:
Please let me know if there are better options, ideally with minimum/safe changes.

Reply:
Hmm right, I was trying to type safe everything but now we are forced to go to the opposite direction.

Reply:
Well, we can discuss and define some rules about where the strong types are good and required and where not. For example, an idea is for the UI layer to not know about "internal" types that it doesn't care about, the UI models should represent what the user sees and no more than that.